Functions/Common_Utils/TransformUtils.cs

```diff
@@ -24,11 +23,29 @@ public class TransformUtils
         /// <returns></returns>
         public static async Task<Transform> CreateEncodingTransform(IAzureMediaServicesClient client, ILogger log, string resourceGroupName, string accountName, string transformName, string builtInPreset)
         {
-            // Does a transform already exist with the desired name? Assume that an existing Transform with the desired name
-            // also uses the same recipe or Preset for processing content.
-            Transform transform = client.Transforms.Get(resourceGroupName, accountName, transformName);
+            bool createTransform = false;
+            Transform transform = null;
 
-            if (transform == null)
+            try
+            {
+                // Does a transform already exist with the desired name? Assume that an existing Transform with the desired name
+                // also uses the same recipe or Preset for processing content.
+                transform = client.Transforms.Get(resourceGroupName, accountName, transformName);
```

Comment:
Maybe we should not do this check and just do a CreateOrUpdate on the Transform... if it changed, it can then be updated... if it does not change - it's just an Update right?  

I've messed up a lot on this issue in the past.

Reply:
Yes, the Get is optional. It is possible to do a CreateOrUpdate. I thought of that but let the code as is.
Two reasons
- do we want to always update the transform ? Perf impact ? Corruption of an existing transform ?
- a customer may prevent a transform to be overridden. In that case, they have the structure to stop the code if an existing transform is detected.